packages/desktop/src/renderer/components/widgets/channels/ChannelInput/ChannelInput.tsx

```diff
@@ -470,8 +476,20 @@ export const ChannelInputComponent: React.FC<ChannelInputProps> = ({
                   html={sanitizedHtml}
                   onChange={onChangeCb}
                   onKeyDown={onKeyDownCb}
-                  onPaste={(e) => {
+                  onPaste={async (e) => {
                     e.preventDefault()
+
+                    const files = e.clipboardData.files
+                    for (let i = 0; i < files.length; i++) {
+                      const fileNameSplited = files[i].name.split('.')
+                      const fileExtension = fileNameSplited[fileNameSplited.length - 1]
```

Comment:
Can't you use `path` methods instead? It will be consistent with the rest of our paths data

```
> const a = 'image.png'
undefined
> path.basename(a, path.extname(a))
'image'
> path.extname(a)
'.png'
```